src/GameLoop.tsx

```diff
@@ -402,7 +402,7 @@ export function GameLoop() {
             else {
                 stateHandler.send("DONE");
             }
-        } else if ("spawningBlock" == stateHandler.state.value) {
+        } else if ("spawningBlock" === stateHandler.state.value) {
```

Comment:
If it's a small change, and it's not important to keep the order of commits with respect to the discussion and comments on the PR, it's better to rebase.

Reply:
Merging is not that bad, though.

Reply:
As in `git rebase main`? You're right. I'll do that next time